config/src/main/java/tech/pegasys/pantheon/config/IbftConfigOptions.java

```diff
@@ -39,4 +41,12 @@ public int getBlockPeriodSeconds() {
   public int getRequestTimeoutSeconds() {
     return ibftConfigRoot.getInteger("requesttimeoutseconds", DEFAULT_ROUND_EXPIRY_SECONDS);
   }
+
+  public int getSeenMessageQueueSize() {
```

Comment:
s/getSeenMessageQueueSize/getSeenMessageQueueLimit

QueueSize could be mistaken as being the current queue size

Reply:
I like limit better. Still don't like these config names though regardless of size or limit suffix. So if have any suggestions...